fleetspeak/src/inttesting/frr/frr.go

```diff
@@ -424,28 +424,33 @@ func (s *MasterServer) AllRequests(id common.ClientID) []int64 {
 	return r
 }
 
-// CompletedRequests returns a list of requests made to a client which have been
-// completed.
-func (s *MasterServer) CompletedRequests(id common.ClientID) []int64 {
```

Comment:
Let's keep this (non-rpc) method as well, under a different name. And let's have the new rpc method call the internal one. This way we don't have to deal with the RPC boilerplate for internal uses of CompletedRequets.

Maybe something like:

func ... completedRequests(...) ... // will it still be visible in the tests though? otherwise some other public name
func ... CompletedRequests(...) ... { calls completedRequests }

(Same for the second new RPC method)

Reply:
If we use completedRequests(...) it's visible from frr_test.go but not visible from other test files. It's called only from frr_test.go, however integrationtest/frr.go uses CreateHunt (it's now called CreateBroadcastRequest), so I made both non-rpc methods public